lib/modules/manager/nix/schema.ts

```diff
@@ -0,0 +1,41 @@
+import { z } from 'zod';
+import { Json } from '../../../util/schema-utils';
+
+export const InputType = z.enum([
```

Comment:
```suggestion
const InputType = z.enum([
```
don't export if not used